cohort-util/src/main/java/com/ibm/cohort/valueset/ValueSetUtil.java

```diff
@@ -46,7 +49,8 @@ public static void validateArtifact(ValueSetArtifact valueSetArtifact){
 			}
 	}
 
-	public static ValueSetArtifact createArtifact(InputStream is) throws IOException {
+	//todo should new map be passed in here, or somewhere else? do we want to limit this to one custom code system, or multiple? If there are multiple
```

Comment:
Pedantic: Is this an active TODO, or leftover?

Reply:
leftover, killing